src/ml/hac.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
```

Comment:
Can we delete these duplicate files and the example output images?

Reply:
yes